toolchains/built_toolchains.bzl

```diff
@@ -403,16 +404,17 @@ def _cmake_toolchain(version):
 
     fail("Unsupported cmake version: " + str(version))
 
-def _make_toolchain(version):
-    native.register_toolchains(
-        "@rules_foreign_cc//toolchains:built_make_toolchain",
-    )
+def _make_toolchain(version, register_toolchains):
+    if (register_toolchains):
```

Comment:
nit: Why is this wrapped in braces?

Reply:
Context switching between too many languages!

Reply:
```suggestion
    if register_toolchains:
```